examples/api/lib/cupertino/nav_bar/cupertino_sliver_nav_bar.0.dart

```diff
@@ -0,0 +1,105 @@
+      title: _title,
+      home: CupertinoNavBarSample(),
+    );
+  }
+}
+
+class CupertinoNavBarSample extends StatelessWidget {
+  const CupertinoNavBarSample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      // A ScrollView that creates custom scroll effects using slivers.
+      child: CustomScrollView(
+        // A list of sliver widgets.
+        slivers: <Widget>[
+          const CupertinoSliverNavigationBar(
+            leading: Icon(CupertinoIcons.person_2),
+            // This title is visible in both collapsed and expanded states.
+            // when `middle` parameter is omitted, it takes the `largeTitle`.
+            largeTitle: Text('Contacts'),
+            trailing: Icon(CupertinoIcons.add_circled),
+          ),
+          // This widget fills the remaining space in the viewport.
+          // Drag the scrollable area to collapse CupertinoSliverNavigationBar.
+          SliverFillRemaining(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: <Widget>[
+                const Text('Drag me up', textAlign: TextAlign.center),
+                CupertinoButton.filled(
+                  onPressed: () {
+                    Navigator.push(context, CupertinoPageRoute<Widget>(builder: (BuildContext context) {
+                      return const NextPage();
+                    }));
+                  },
+                  child: const Text('Go to Next Page'),
+                )
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class NextPage extends StatelessWidget {
+  const NextPage({ Key? key }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Brightness brightness = CupertinoTheme.brightnessOf(context);
+    return  CupertinoPageScaffold(
+      child: CustomScrollView(
+        slivers: <Widget>[
+          CupertinoSliverNavigationBar(
+            backgroundColor: CupertinoColors.systemYellow,
+            border: Border(
+              bottom: BorderSide(
+                color: brightness == Brightness.light
+                  ? CupertinoColors.black
+                  : CupertinoColors.white,
+              ),
+            ),
+            // The middle widget is visible in both collapsed and expanded states.
+            middle: const Text('Contacts Group'),
+            // When `middle` parameter is implemented, larget title is only visible
+            // when the CupertinoSliverNavigationBar is fully expanded.
+            largeTitle: const Text('Family'),
+          ),
+          SliverFillRemaining(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: const <Widget>[
+                Text('Drag me up', textAlign: TextAlign.center),
+                // When `leading` parameter is omitted on a route that has a previous page,
```

Comment:
```suggestion
                // When the "leading" parameter is omitted on a route that has a previous page,
```

These aren't Dartdoc comments, so use regular quotes instead.